redaxo/src/core/tests-visual/visual-record.js

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
Ich überlege, ob der Ordner vielleicht eher in den Root gehört?
Er wird ja auch Bilder der Addons enthalten, und ist allgemein ja eher ein Tool hier fürs Repo, oder?
(Und soll zum Beispiel auch nicht ins Release)

Oder vielleicht sogar mit in `.github`?
Oder machen wir ggf. einen neuen Ordner, wo alle Dateien rein kommen, die so von den Tools verwendet werden? Also die Dateien hier, plus auch die zusätzlichen für phpstan und psalm (nicht die haupt-configs, die sollten für die einfachere Nutzung im Root bleiben).

Reply:
da bin ich komplett offen.. will es aktuell erstma leinigermaßen zuverlässig ans laufen bekommen

Reply:
bin hier auch eher bei der variante des `.github` ordners.

zumal sind ja in `index.png` screen jetzt schon inhalte von structure, weshalb der core ordner IMO net passt.